liquibase-core/src/main/java/liquibase/util/FilenameUtil.java

```diff
@@ -64,6 +65,28 @@ public static String concat(String basePath, String fullFilenameToAdd) {
         return normalize(Paths.get(basePath, fullFilenameToAdd).toString());
     }
 
+    /**
+     * Joins the specified path elements with a single forward slash.
+     * Existing forward slashes are not duplicated. The operating system is not consulted for path separator, a forward
+     * slash is always used.
+     *
+     * @param  path            Elements of the path
+     */
+    public static String concatWithForwardSlash(String... path) {
```

Comment:
Are the new `concat` methods on Resource the better version of this?

Reply:
Removing this as it is not needed.